terraform/environments/nomis/providers.tf

```diff
@@ -78,3 +78,7 @@ provider "aws" {
 #   }
 # }
 ######################### Run Terraform Plan Locally Only ##################################
+
+provider "github" {
```

Comment:
Just for simplicity's sake could you please add this to run to the run via ci/cd and run locally block please? When changing this file from local to automated, it might be too easy to comment this out accidentally.